expressEngine/lib/successHandler/successHandler.js

```diff
@@ -1,6 +1,3 @@
-export default function successHandler(message, data = null) {
-  return ({
-    data,
-    message,
-  });
-}
+import success from '../../cms/common/success';
+
+export default (message = success.defaultSuccessMsg, data = null) => ({ data, message });
```

Comment:
why are you using success handler if you are not modifying the arguments, you are returning the passed argument.
The only use I see here is that you are setting the default value which can be done in the parser itself.